commandline/src/main/java/tech/pegasys/web3signer/commandline/PicoCliAwsKmsParameters.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline;
+
+import tech.pegasys.web3signer.common.config.AwsAuthenticationMode;
+import tech.pegasys.web3signer.signing.config.AwsParameters;
+
+import java.net.URI;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import picocli.CommandLine.Option;
+
+public class PicoCliAwsKmsParameters implements AwsParameters {
+  public static final String AWS_KMS_ENABLED_OPTION = "--aws-kms-enabled";
+  public static final String AWS_KMS_AUTH_MODE_OPTION = "--aws-kms-auth-mode";
+  public static final String AWS_KMS_ACCESS_KEY_ID_OPTION = "--aws-kms-access-key-id";
+  public static final String AWS_KMS_SECRET_ACCESS_KEY_OPTION = "--aws-kms-secret-access-key";
+  public static final String AWS_KMS_REGION_OPTION = "--aws-kms-region";
+  public static final String AWS_ENDPOINT_OVERRIDE_OPTION = "--aws-endpoint-override";
```

Comment:
nit: should this be aws-kms-endpoint-override or is endpoint the same for SM and KMS?

Reply:
endpoint it the same for both